src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -55,6 +56,7 @@ export function SidebarNavSign({ blockedReason, intl, status, targetingApi, ...r
             <Tooltip isDisabled={isTargeted} position={TooltipPosition.MIDDLE_LEFT} text={tooltipMessage}>
                 <PlainButton aria-label={label} className={buttonClassName} isDisabled={isSignDisabled} {...rest}>
                     <BoxSign28 className="bcs-SidebarNavSign-icon" />
+                    <Sign16 width={20} height={20} className="bcs-SidebarNavSign-icon--greyscale" />
```

Comment:
nit: majority of spelling in this repo is `gray` vs `grey`

```suggestion
                    <Sign16 width={20} height={20} className="bcs-SidebarNavSign-icon--grayscale" />
```

Reply:
Got it. Going with `gray`